modules/face/include/opencv2/face/face_alignment.hpp

```diff
@@ -0,0 +1,145 @@
+and on any theory of liability, whether in contract, strict liability,
+or tort (including negligence or otherwise) arising in any way out of
+the use of this software, even if advised of the possibility of such damage.*/
+#ifndef __OPENCV_FACE_ALIGNMENT_HPP__
+#define __OPENCV_FACE_ALIGNMENT_HPP__
+#include "opencv2/core.hpp"
+#include "opencv2/objdetect.hpp"
+#include "opencv2/imgproc.hpp"
+#include <iostream>
+#include <string>
+#include <sstream>
+#include <vector>
+#include <fstream>
+#include <queue>
+#include <algorithm>
+#include <fstream>
+
+namespace cv{
+namespace face{
+class FacemarkKazemi
+{
+public:
+        /** @returns The number of landmarks detected in an image */
+        virtual int getNumLandmarks() const = 0;
+        /** @returns The number of faces detected in an image */
+        virtual int getNumFaces() const = 0;
+        /** @brief This function extract the data for training from .txt files which contain the corresponding image name and landmarks.
+        *Each file's first line should give the path of the image whose
+        *landmarks are being described in the file.Then in the subsequent
+        *lines there should be coordinates of the landmarks in the image
+        *i.e each line should be of the form "x,y"(ignoring the double quotes)
+        *where x represents the x coordinate of the landmark and y represents
+        *the y coordinate of the landmark.
+        *
+        *For reference you can see the files as provided in the
+        *<a href="http://www.ifp.illinois.edu/~vuongle2/helen/">HELEN dataset</a>
+        * @param filename A vector containing name of the .txt files.
+        * @param trainlandmarks A vector that would store shape or landmarks of all images.
+        * @param trainimages A vector which stores the images whose landmarks are tracked
+        * @returns A boolean value
+        */
+        virtual bool getData(std::vector<String> filename,std::vector< std::vector<Point2f> > & trainlandmarks
+                ,std::vector<String> & trainimages)=0;
+        /** @brief This function gets the relative shape of the face scaled and centered
+        according to the bounding rectangle of the detected face.
+        * @param face A vector storing rectangles of all  faces in the current image.
+        * @param initialshape A vector storing initial shapes of all the faces in the current image.
+        * @returns A boolean value
+        */
+        virtual bool getMeanShapeRelative(std::vector<Rect> face,std::vector< std::vector<Point2f> > & initialshape)=0;
+        /** @brief This function gets the bounding rectangle of all the faces in an image.
+        * choose 0 to select HAAR cascade classifier.
+        * choose 1 to select LBP cascade classifier.
+        * @param src  stores the image in which faces have ti be detected
+        * @param facep  stores the bounding boxes of the face whose initial shape has to be found out
```

Comment:
Stores in both lines with capital, we started all sentences with capital and lets end all with a dot, just to make it uniform!